static/fetch_lib.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# shellcheck disable=2034,2059
+true
+# see https://github.com/koalaman/shellcheck/wiki/Directive
+
+
+mkdir -p /var/scripts
+if ! [ -f /var/scripts/lib.sh ]
+then
+    if ! curl -so /var/scripts/lib.sh https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh 
+    then
+        printf "The local lib isn't available and you don't have internet access. You cannot run this script"
+        exit 1
+    fi
+elif test "$(find /var/scripts/lib.sh -mmin +30)"
+then
+    curl -so /var/scripts/lib.sh https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh
+fi
+
+# shellcheck source=lib.sh
+source /var/scripts/lib.sh &>/dev/null
```

Comment:
@enoch85 I just redid my whole version. I hope it is better understandable now.

Reply:
We pushed at the same time. Check the latest.

Reply:
I think this does the same but is much more simplistic which I really like. :)

Reply:
@enoch85 I think the biggest advantage, is that we don't have to check the internet each time. We just try to fetch it and if it doesnt work, the internet isn't working correctly. If the file exists, we just try to fetch a new one (if it is older than 30mins in this case to have the file always up do date. If that doesnt work here, it is no problem since we just use the local lob either way

Reply:
Please join the chat: https://cloud.hanssonit.se/call/3ejfaazp

Reply:
ping @szaimen